### PR TITLE
Add materials to schedule

### LINKED ISSUE #73: Integrate slide links into the schedule
The workshop schedule document should be updated to include links to individual sets of slides throughout the day. This can either be links to PDFs that end up in this repo, or this can be read-only links to google slides themselves.

### PR DESCRIPTION
**Pull request by sjspielman**
Closes #73 

This PR (_not yet ready for review_) adds links into the schedule, including slides and UNIX cheatsheets.

### CODE REVIEW

**Comment by sjspielman** [comment]
This is ready for review @jashapiro , but we may not want to _merge_ until Thursday?
Please check that the links are all correct and that the schedule times look ok! Note that this PR also added/ran a script to render and copy UNIX cheatsheets into `docs/` (though the quick cheatsheet is rendered manually as described).

Tagging @Jen-OMalley for context as this relates to internal schedule.

File: workshop_materials/cheatsheets/process_unix_cheatsheets.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -euo pipefail
+
+# Note that the `unix_quick_reference.pdf` should be created manually by 
+# by exporting the `unix_quick_reference.md` as a PDF from the 
+# MacDown application on macOS _before_ running this script.
+
+# Directory where cheatsheets will be copied to
+DEST_DIR=../../docs/
```
Comment (jashapiro) [inline workshop_materials/cheatsheets/process_unix_cheatsheets.sh:9]: Let's maybe put these in a subdirectory?
```suggestion
DEST_DIR=../../docs/resources/
mkdir -p $DEST_DIR
```

File: docs/workshop-schedule.md [hunk 2]
```diff
@@ -3,18 +3,18 @@ title: Workshop Schedule
...
-| 2:00 - 2:30 pm   | The "stage/commit/push" git cycle <br> `.gitignore` files |
-| 2:30 - 2:45 pm   | Coffee break |
-| 2:45 - 3:15 pm   | Scripts, notebooks, and literate programming |
-| 3:15 - 3:45 pm   | Managing package versions with `renv` <br> Working with branches in git |
-| 3:45 - 4:15 pm   | Documenting your projects |
+| 11:00 - 11:30 am | [Introduction to UNIX and the command line](https://docs.google.com/presentation/d/1WPXkItJZEUXMY20cLrdMXHiBC2PyunR14RUSDg4nfIc/edit?usp=sharing) <br> [Resource: UNIX Reference](./unix_reference.html) <br> [Resource: UNIX Cheatsheet](./unix_quick_reference.pdf) |
+| 11:30 - 12:00 pm | [Introduction to Git; Forking and cloning a repository](https://docs.google.com/presentation/d/1eiGZA4PYBKJx5HDCo3UDOAB7q415gg96TehxilCHwlA/edit?usp=sharing) |
+| 12:00 - 1:00 pm  | Lunch |
+| 1:00 - 1:45 pm   | [Shell scripting](https://docs.google.com/presentation/d/1SDUyYVNgvDDRodVqmDQPVQ5wnjQesWfBTg0EAcdbcSo/edit?usp=sharing) |
+| 1:45 - 2:15 pm   | [The Git Stage/Commit/Push workflow and `.gitignore` files](https://docs.google.com/presentation/d/1_YckNhAkp_82PKR6PGS5SdaKDgoueYVTXaPi5pQV9ik/edit?usp=sharing) |
```
Comment (jashapiro) [inline docs/workshop-schedule.md:15]: Just noting this deck had permission that allowed comments; I updated it to view only.

**Review by jashapiro — DISMISSED** [review]
Looks good!

Just one minor suggestion about directory organization.

File: docs/workshop-schedule.md (hunk 2 above)
Comment (sjspielman) [reply docs/workshop-schedule.md:15]: !! good catch.

**Comment by jashapiro** [comment]
One other thing for this PR might be to add a link to PDF of all slides, or maybe that goes in #34?

**Pull request merged**